package.json

```diff
@@ -18,6 +18,7 @@
     "start:dev": "lerna run --parallel start:dev",
     "test": "lerna run --parallel test",
     "start:docker": "docker-compose up",
-    "test:docker": "docker-compose -f docker-compose.override.yml up"
+    "test:docker": "docker-compose -f docker-compose.override.yml up",
+    "configure:env" : "node configure-env.js"
```

Comment:
this can be a pre-start step for everything

Reply:
Sure, currently I have set up to run this script as a pre-start step by using `&&` symbol in `start`, `start:dev` and `start:docker`. I do have a question about whether it is better to use the "pre" scripts for each of them. (i.e. having `"prestart" : "node configure-env.js"` and `"start": "lerna run --parallel start"` rather than `"start": "node configure-env.js && lerna run --parallel start"`

Reply:
I think `prestart` is better